tests/examples/externals/definitions.py

```diff
@@ -0,0 +1,47 @@
+from dusk.script import *
+
+# A `definitions.py` could be one way to naturally introduce symbols that can
+# be imported and used in dusk stencils.
+
+regions = Regions(
+    # Here we specify which magic integers to use. This enables us to quickly
+    # prototype new features. Once SIR has built-in support for this, it can
+    # be changed.
+    lb=1,
+    nudging=2,
+    interior=3,
+    halo=4,
+)
+
+# string encoding
+another_regions = HorizontalRegions(
+    lb="lb", nudging="nudging", interior="interior", halo="halo"
+)
+# compact string encoding
+regions_again = HorizontalDomains("lb", "nudging", "interior", "halo")
+
+# with python's value unpacking, we can introduce quick shortcuts as well
+lb, nudging, interior, halo = regions = Domains("lb", "nudging", "interior", "halo")
+
+# can also skip `regions`
+lb, nudging, interior, halo = Domains("lb", "nudging", "interior", "halo")
+
+# Now these external symbols are well integrated in Python. Dusk can import them
+# and find out what to do based on the type of the objects.
+# E.g., encode `Regions` through the corresponding magic integers/strings.
+
+# Other kinds of external symbols can be easily supported as well.
+k_flat = VerticalLevel("k_flat")
```

Comment:
This is a bit trickier. Firstly, no implementation has been planned for dusk or dawn so far.

But the bigger issue is that we have to do two things:
1. introduce a new symbol (e.g., `k_flat`)
2. _connect_ that symbol somehow to what it will _mean_ (the `"k_flat"` part).

This is similar to python's `TypeVar` which suffers from the same redundancy: https://docs.python.org/3/library/typing.html#typing.TypeVar

We could similarly do something like:

```python
vertical_markers = Intervals("k_flat", "ground", "heaven", ...)
```

But if you want to introduce only one single vertical marker, I'm not sure how to resolve this redundancy. Still, this is only an example and we don't really know how we'll handle vertical interval markers yet.